resources/jscomposition/system/table/cell/StatusCell.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div class="tw-flex tw-justify-center">
+    <span
+      :class="`tw-inline-flex tw-items-center tw-text-xs tw-border-1 tw-rounded-lg
+        tw-bg-${color}-100 tw-px-2 tw-py-1 tw-text-${color}-500`">
+      {{ label }}
+    </span>
+  </div>
+</template>
+<script>
+import { defineComponent, computed } from "vue";
+
+export const statuses = {
+  DRAFT: {
+    color: "red",
+    label: "Draft",
+  },
+  CANCELED: {
+    color: "red",
+    label: "Canceled",
+  },
+  COMPLETED: {
+    color: "blue",
+    label: "Completed",
+  },
+  ERROR: {
+    color: "red",
+    label: "Error",
+  },
+  IN_PROGRESS: {
+    color: "green",
+    label: "In progress",
+  },
+};
```

Comment:
Use these colors primary, warning, danger, success.
In order to keep the compatibility with customizeUI

Reply:
The status colors in the new design are not equals that old status colors

Reply:
Draft is no red